config.template/faf-rust-replayserver/faf-rust-replayserver.yml

```diff
@@ -0,0 +1,22 @@
+# See faf-rust-replayserver repository, docs/usage.rst for documentation.
+server:
+        port: 15000
+        prometheus_port: 8011
+        worker_threads: 4
+        connection_accept_timeout_s: 21600
+database:
+        pool_size: 8
+        host: faf-db
+        port: 3306
+        user: faf-replayserver
```

Comment:
For the time being please use faf-aio-replayserver.
As is it won't work because it's missing in `scripts/init-db.sh` and having 2 users for the same thing sound unnecessary